platformio.ini

```diff
@@ -87,5 +87,16 @@ custom_openshock.chip_variant = N8R8
 build_flags =
 	-DOPENSHOCK_LED_GPIO=21
 
+; https://github.com/nullstalgia/OpenShock-Hardware/tree/main/Core
+[env:OpenShock-Core-V1]
+board = esp32-s3-devkitc-1 ; builtin
+custom_openshock.chip = ESP32-S3
+custom_openshock.chip_variant = N8R2
+build_flags =
+	-DOPENSHOCK_LED_WS2812B=48
+	-DOPENSHOCK_LED_GPIO=35
```

Comment:
Two leds defined? this is ok as long as it compiles, but idk if this is supported or why we would need it 🤔

Reply:
I added support for it, the GPIO one just turns into a simple "all is ok" LED, as described in the PR above.